spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -66,3 +66,6 @@ definitions:
                   values:
                       - 0: System Healthy
                       - 1: An error has occurred
+        - protocol:
```

Comment:
@mookerji are we using a version string? If we we're packing the version elements (major, minor, sub-minor) into the reserved field, that might be possible - but if we're using an opaque string, that doesn't seem like it will work.

/cc @henryhallam